backend-core-business-spring-impl/src/main/java/com/flowingcode/backendcore/service/ConstraintSpecification.java

```diff
@@ -13,6 +13,12 @@
 import lombok.AccessLevel;
 import lombok.RequiredArgsConstructor;
 
+/**
+ * Spring Data JPA {@link org.springframework.data.jpa.domain.Specification} that applies a model {@link com.flowingcode.backendcore.model.Constraint}.
```

Comment:
The following identifiers must never be linked and should always be wrapped in {@code}:
- The names of the direct supertype and any implemented interfaces of the documented type (in a type-level comment)
https://github.com/FlowingCode/DevelopmentConventions/blob/94fe5e464b0d91cb56542aeb098304c92f88b2ec/code-style.md#when-to-use-code